index.html

```diff
@@ -59,227 +60,218 @@ <h1>I'm Abdi</h1>
+                    <li>  <a href="#">   <button type="button" id="button_link">bootstrap</button></a></li>
+                    <li> <a href="#"> <button type="button" id="button_link">Ruby</button></a></li>
+                 </ul>
+               </div>
+                 <div class="seeProjectCards">
+                   <a href="#"><button type="button" id="button_linkProj">See Project</button></a>  
+                 </div>
+             </div>
 
+          <!--Card7-->
       
-            <div class="cards card7">
+             <div class="cards card7">
                 <h3 class="cards-header">Profesional Art <br>Printing Data</h3>
-                <div class="cards_description">
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's 
-                        standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-                </div>
-              
+                 <div class="cards_description">
+                   <p>A daily selection of privately personalized reads; no accounts or sign-ups 
+                     required. has been the industry's 
+                     standard dummy text ever since the 1500s, 
+                     when an unknown printer took a standard dummy text.
+                  </p>
+               </div>
                 <div class="card_buttons">
-                    <ul>
-                        <li> <a href="#">  <button type="button" id="button_link">html</button></a></li>
-                                        <li>  <a href="#">   <button type="button" id="button_link">bootstrap</button></a>
-                                        </li>
-                                        <li> <a href="#">  <button type="button" id="button_link">Ruby</button></a></li>
-                        </ul>
-                </div>
-                <div class="seeProjectCards">
-                    <a href="#"><button type="button" id="button_linkProj">See Project</button></a>  
-              </div>
-            
-              
-            
+                  <ul>
+                    <li> <a href="#">  <button type="button" id="button_link">html</button></a></li>
+                    <li>  <a href="#">   <button type="button" id="button_link">bootstrap</button></a></li>
+                    <li> <a href="#"> <button type="button" id="button_link">Ruby</button></a></li>
+                 </ul>
+               </div>
+                 <div class="seeProjectCards">
+                   <a href="#"><button type="button" id="button_linkProj">See Project</button></a>  
+                 </div>
+             </div>
             </div>
-           
-
-
         </div>
-       
-       
+</section>
+
+        <!--Completed Works Section the below is About me section-->
 
-    </section>
     <div class="about_Me">
```

Comment:
Kindly note that your project has borders and margins not in the figma design. It is important that your project maches the figma design as much as possible
This is how your project currently is:
<img width="1636" alt="Screenshot 2022-09-30 at 22 50 23" src="https://user-images.githubusercontent.com/48546992/193351492-c6ff95db-8a3a-47d2-b11a-db8c8449900f.png">

The figma design below has no morgins. It fills the full screen for the about page
<img width="715" alt="Screenshot 2022-09-30 at 23 30 32" src="https://user-images.githubusercontent.com/48546992/193351777-9e413447-ef62-4564-ad19-525c74c7aec8.png">